android/CouchbaseLite/src/main/java/com/couchbase/lite/DatabaseConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package com.couchbase.lite;
+
+import android.content.Context;
+
+public class DatabaseConfiguration extends BaseDatabaseConfiguration{
+    private Context context;
+
+    public DatabaseConfiguration() {
```

Comment:
So you are saying `DatabaseConfiguration` constructor only support `DatabaseConfiguration(Context)` for Android? To simplify the API, this could be a reasonable solution.